### PR TITLE
3 (Create & Read Shipment)

### PR DESCRIPTION
**Pull request by Fube**
# Jira Issue: [Create & Read Shipment](https://champlainsaintlambert.atlassian.net/browse/PLUMBERS-3)

### Changes
- Added Shipment entity
- Added migrations scripts for Shipment
- Added RESTlike endpoints for CR on Shipment
- Added facets for Stock entity on Algolia

#### Shipments table
image

#### Create shipment form without a 'to' set
image

#### Create shipment form with a 'to' set
image

### CODE REVIEW

**Review by zellyk — APPROVED** [review]
Very good, unit testing, E2E testing, as well as screenshots. 👍🏼

File: UI/types/package.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
     "name": "@lepine/ui-types",
-    "version": "1.0.1"
+    "version": "1.0.2"
```
Comment (MaxGrabs) [inline UI/types/package.json:3]: do i need to change anything on my end?

**Review by MaxGrabs — COMMENTED** [review]
very nicely done!

File: UI/types/package.json (hunk 1 above)
Comment (Fube) [reply UI/types/package.json:3]: Nope

File: API/src/main/java/com/lepine/transfers/data/shipment/ShipmentStatusLessCreatedByLessUuidLessDTO.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import javax.validation.constraints.Size;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(toBuilder = true)
+public class ShipmentStatusLessCreatedByLessUuidLessDTO {
```
[the remark this answers is not in the document]
Comment (Fube) [reply API/src/main/java/com/lepine/transfers/data/shipment/ShipmentStatusLessCreatedByLessUuidLessDTO.java:21]: It is missing the `UUID createdBy` field

**Pull request merged**